custom_components/sector/coordinator.py

```diff
@@ -30,7 +30,7 @@ def __init__(self, hass: HomeAssistant, entry: SectorAlarmConfigEntry) -> None:
         """Initialize the coordinator."""
         self.hass = hass
         self.api = SectorAlarmAPI(
-            hass,
+            hass=self.hass,
```

Comment:
```suggestion
            hass=hass,
```
Not sure why this changed